pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.openhab.addons</groupId>
     <artifactId>org.openhab.addons.reactor</artifactId>
-    <version>2.5.0</version>
+    <version>2.5.1-SNAPSHOT</version>
```

Comment:
This one might be ok to change to 2.5.1-SNAPSHOT as it is about the addons.
So let's have a try - but please remove the other changes.